dragula.js

```diff
@@ -605,4 +613,66 @@ function getCoord (coord, e) {
   return host[coord];
 }
 
+function getScrollContainer(node) {
+  if (node === null) { return null; }
+  if (node.scrollHeight > node.clientHeight) {
+    return node;
+  } else {
+    if (!/(body|html)/i.test(node.parentNode.tagName)) {
```

Comment:
I don't understand why I need a variable for this. Why would you want to hold on to the reference if it's not going to be used unless we're scrolling?